modules/k8s/tanka/new-app.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+APP_NAME=$1
+
+NAMESPACE=${NAMESPACE:-""}
+OWNER=${OWNER:-""}
+IMAGE_REPOSITORY=${IMAGE_REPOSITORY:-""}
+IMAGE_TAG=${IMAGE_TAG:-""}
+
+echo ""
+
+if [ -z ${APP_NAME} ]; then
+  echo "Must specify an application name"
+  exit 1
+fi
+if [ -d "lib/${APP_NAME}" ] || [ -d "environments/${APP_NAME}" ]; then
+  echo "Application already exists by the name '${APP_NAME}'"
+  exit 1
+fi
+
+if [ "${NAMESPACE}" = "" ]; then
+  namespaces=$(tk env list --json | jq -r '.[].spec.namespace' | sort | uniq)
```

Comment:
This might fail in repos like the `shared-services` one.

We might just want a print statement at the end along the lines of:

_Make sure you review and edit hte generated ./lib/$app/base.libsonnet_ file

Reply:
That's why I make the script prompt for a namespace if the search returns more than one.

but I like the idea of a final statement to review the output